InsiteTeamTask.Data/Repositories/DataRepository.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using InsiteTeamTask.Data.MockData;
+using InsiteTeamTask.Data.Models;
+
+namespace InsiteTeamTask.Data.Repositories
+{
+    internal class DataRepository : IDataRepository
```

Comment:
This class implements an empty interface and isn't used anywhere - it isn't even added to the app's service collection